tests/regression/test_ensembleparallelism.py

```diff
@@ -25,6 +25,24 @@
             pytest.param(False, id="nonblocking")]
 
 
+def parallel_assert(assertion, subset=None, msg=""):
+    """ Move this functionality to pytest-mpi
+    """
+    if subset:
+        if MPI.COMM_WORLD.rank in subset:
+            evaluation = assertion()
+        else:
+            evaluation = True
+    else:
+        evaluation = assertion()
+    all_results = MPI.COMM_WORLD.allgather(evaluation)
+    if not min(all_results):
```

Comment:
```suggestion
    if not all(all_results):
```
`Comm.allgather` doesn't convert bools to ints.